autotag

```diff
@@ -48,6 +48,19 @@ git fetch --tags
 echo 'Finding most recent tag...'
 
 LAST_TAG=$(git describe --abbrev=0 --tags 2>/dev/null)
+
+# Ensure that tag is not created on same commit as current branch commit
+if [[ ! -z "$LAST_TAG" ]]; then
+  CURRENT_BRANCH=$(git branch --show-current)
+  BRANCH_HASH=$(git rev-list -n 1 $CURRENT_BRANCH)
```

Comment:
If you use `rev-parse` it already gets one commit so you don't have to set it.

https://michaelcurrin.github.io/dev-cheatsheets/cheatsheets/version-control/git/advanced.html

```suggestion
  BRANCH_HASH=$(git rev-parse $CURRENT_BRANCH)
```